packages/wallets/src/hooks/useInputATMFormatter.ts

```diff
@@ -123,7 +124,11 @@ const useInputATMFormatter = (inputRef: React.RefObject<HTMLInputElement>, initi
                 isPasting.current = false;
                 if (!isRewriting.current) return;
 
-                const pastedValue = Number(unFormatLocaleString(clipboardContent.current, locale));
+                const pastedValueUnformatted = unFormatLocaleString(clipboardContent.current, locale);
```

Comment:
For the inputs, given they are getting more and more complex, it would be especially valuable to have tests (as documentation for business rules around them). 

From my experience with inputs of that type, soon we gonna be running in circles - one fix is gonna be changing some previous behavrious, then next fix is gonna be changing some other behavrious or reverting previous changes and without tests there will be just endless stream of fixes and changes :D